frontend/src/features/projects/list/ProjectListView.tsx

```diff
@@ -168,20 +168,11 @@ const ProjectListView: React.FC<IProps> = ({ filterable, title, mode }) => {
     const ReviewWorkflowStatuses = Object.keys(ReviewWorkflowStatus).map(
       (k: string) => (ReviewWorkflowStatus as any)[k],
     );
-    if (
-      (row.statusCode === ReviewWorkflowStatus.PropertyReview ||
-        row.statusCode === ReviewWorkflowStatus.ExemptionReview) &&
-      keycloak.hasClaim(Claims.ADMIN_PROJECTS)
-    ) {
-      history.push(`/dispose/projects/assess/properties?projectNumber=${row.projectNumber}`);
-    } else if (ReviewWorkflowStatuses.includes(row.statusCode)) {
-      if (
-        keycloak.hasClaim(Claims.ADMIN_PROJECTS) &&
-        row.statusCode !== ReviewWorkflowStatus.Denied
-      ) {
-        history.push(`/dispose/projects/approved?projectNumber=${row.projectNumber}`);
+    if (ReviewWorkflowStatuses.includes(row.statusCode)) {
```

Comment:
This is getting closer to what I want, which is that all project routes can be navigated to based purely on project status.